crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -70,6 +70,7 @@ pub struct EnergyStats {
 
 pub struct ExecutionTimings {
     pub total_duration: Duration,
+    #[expect(unused)] // TODO: do we want to do something with this?
```

Comment:
Seriously, did Rust's tree-shaking / dead-code-elimination change in the past few months?

Reply:
Correct, yeah.